apisix/core/config_etcd.lua

```diff
@@ -545,7 +564,24 @@ local function _automatic_fetch(premature, self)
 
             local ok, err = sync_data(self)
             if err then
-                if err ~= "timeout" and err ~= "Key not found"
+                if string.find(err, err_etcd_unhealthy_all) then
+                    local reconnected = false
+                    while err and not reconnected do
+                        local backoff_duration, backoff_factor, backoff_step = 1, 2, 10
```

Comment:
Will step `10` too large? As 1024 seconds is too long.

Reply:
I'm not so familiar with the production environment. Do you have some recommendations, like 8?

Reply:
Maybe `6` is enough, also, since we are in a timer, keep a Nginx timer living for a long while is not a good practice as it might cause the memory leaky.

Reply:
Thanks for suggestion!
Changed to use outside counter till `32` to avoid keeping nginx timer too long. PTAL @tokers